packages/components/organisms/f-content-cards/src/components/ContentCards.js

```diff
@@ -336,9 +367,20 @@ export default {
             };
         }
     },
-    render () {
-        return this.$scopedSlots.default({
-            cards: this.cards
-        });
+    render (h) {
```

Comment:
Not sure if `h` is a convention, but what does it stand for?

Reply:
It's a convention, mentioned in the quote at the bottom of [this section](https://vuejs.org/v2/guide/render-function.html#JSX):

> **Aliasing createElement to h is a common convention you’ll see in the Vue ecosystem** and is actually required for JSX. Starting with version 3.4.0 of the Babel plugin for Vue, we automatically inject const h = this.$createElement in any method and getter (not functions or arrow functions), declared in ES2015 syntax that has JSX, so you can drop the (h) parameter. With prior versions of the plugin, your app would throw an error if h was not available in the scope.

I figured it would be better to have it b `h` for forward compatibility - but happy to rename back to `createElement` if this would be preferred.

Reply:
Having slept on it, I think the best of both worlds will to keep it as `h`, but add a comment linking to the documentation above, and explaining that it is short for `createElement`, as this is apparently as yet unfamiliar usage within JET.

Reply:
We actually use `h` in the menu project as, like you say, it's a convention. You'll see in React/Preact also.

Reply:
Catching up on this – think that was the best solution @benwhite-deltas. Thanks for the context too – really helpful